spark/src/main/java/org/opensearch/sql/spark/client/SparkJobClient.java

```diff
@@ -19,4 +20,6 @@ String startJobRun(
       String sparkSubmitParams);
 
   GetJobRunResult getJobRunResult(String applicationId, String jobId);
+
+  CancelJobRunResult closeJobRunResult(String applicationId, String jobId);
```

Comment:
why is it called close ? not cancel ?

Reply:
+1

Reply:
Lot of refactoring missed out....thanks for pointing out.